pkgs/applications/editors/emacs/elisp-packages/elpa-packages.nix

```diff
@@ -106,6 +106,30 @@ self: let
         };
       });
 
+      jinx = super.jinx.overrideAttrs (old: {
+        dontUnpack = false;
+
+        nativeBuildInputs = [
+            pkgs.pkg-config
+        ];
+
+        buildInputs = old.buildInputs ++ [ pkgs.enchant2 ];
+
+        postBuild = ''
+          $CC -O2 -Wall -Wextra -fPIC -shared -o jinx-mod.so jinx-mod.c $($PKG_CONFIG --cflags --libs enchant-2)
```

Comment:
```suggestion
          NIX_CFLAGS_COMPILE="$(pkg-config --cflags enchant-2) $NIX_CFLAGS_COMPILE"
          $CC -shared -o jinx-mod.so jinx-mod.c -lenchant-2
```

Reply:
why not using the `--libs` option of pkg-config and using `-lenchant-2` manually?

Reply:
Since stdenv handles -L, I think it's better to not do it manually.

Maybe people more expericed in C and stdenv have some comments. @NickCao @alyssais